### PR TITLE
fix #80650 brandequity.economictimes.indiatimes.com

### LINKED ISSUE #80650: brandequity.economictimes.indiatimes.com
### Issue URL (Annoyance)
 [https://brandequity.economictimes.indiatimes.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbrandequity.economictimes.indiatimes.com%2F)
### Comment
Username: @chirag127
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>
<details>
  <summary>Screenshot 4</summary>

  Screenshot 4
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by stanislav-atr**
#80650

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -149,6 +149,7 @@ mumbailive.com#$#body { overflow: visible !important; padding-right: 0 !importan
 businessnewsdaily.com##div[class^="css-"][class*="-TransparentContainer"] > div[class^="css-"][class*="-FloaterContainer"]
 vestikavkaza.ru##.wraptip
 mail.ru#?#.balloon:upward(1)
+indiatimes.com##.facebook-live
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:152]: not reproduced

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -2175,7 +2175,7 @@ rautemusik.fm#$#body[style^="margin-top:"] { margin-top: 0!important; }
 simply-yummy.de,tuigroup.com##.cc-grower
 fotoimpex.de###os_app_cookiewarn
 dehner.de###global-message-cookie
-computacenter.com##.consent-container
+indiatimes.com,computacenter.com##.consent-container
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:2178]: not reproduced

File: AnnoyancesFilter/sections/annoyances.txt [hunk 3]
```diff
@@ -1832,6 +1832,7 @@ eastbaytimes.com##.newsletter-simple
 eastbaytimes.com##.Mg2-newsletter-inline_notif_mng
 mag.auto.ru##.jrnl-article-subscription
 hobokengirl.com##.newsbox
+indiatimes.com#?#.footer-card-lst > li:has(> div#subcribe_consentPopup_bottom)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1835]: do not block in footers

**Review by Alex-302 — CHANGES_REQUESTED** [review]
push notification is not blocked

**Comment by stanislav-atr** [comment]
> push notification is not blocked

Not reproduced on my side.

**Pull request closed without merging**